### DISCUSSION TITLE
Get drawing size (height width) using macros

### DISCUSSION THREAD

**Discussion by Willena** [ONLYOFFICE/DocumentServer#1707] [Q&A] [answered]
Hi there,

I'm trying to create a small macro that replace an image by another one. 
I've looked at the available function of the macro API (https://api.onlyoffice.com/docbuilder/classlist) and I'm able to change the image by another one. 
I'm adding the new Image using  `Api.CreateImage` that require to set it size in EMU. 

Is there any function or method I can use to get the dimension of the initial image so that it can be forwarded to the  `Api.CreateImage` method ?

**Comment by ShockwaveNN** [comment]
@askonev Please take a look at this question

**Comment by askonev** [comment, marked as answer]
@Willena Hi! There is currently no way to get the dimensions of an image. Created an enhancement in our private repository `#56439`.

**Reply by Willena** [reply]
> Created an enhancement in our private repository #56439.

@askonev Does this part of your message mean that it something that will be available in a future release ? ( Hopefully soon)

**Comment by Willena** [comment]
@ShockwaveNN , @askonev any news on that subject ?

**Reply by askonev** [reply]
Yes, the methods are implemented and added to the 7.2 release.
